core/src/main/scala/org/apache/spark/api/python/StreamingPythonRunner.scala

```diff
@@ -73,7 +72,10 @@ private[spark] class StreamingPythonRunner(func: PythonFunction, connectUrl: Str
     // Send sessionId
```

Comment:
I really prefer to match the implementation to https://github.com/apache/spark/pull/41948 first. Otherwise, it's difficult to review, and would be more difficult to refactor later.